spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] **[OPTIONAL]** Tests of the `type: :controller` are considered obsolete in RSpec team. You should consider using `type: :request` instead.

  [Link to notice](https://stackoverflow.com/a/46500842)

Reply:
moved to the resource folder